roles/ngi_reports/tasks/dependencies.yml

```diff
@@ -30,10 +30,13 @@
   template: src="texlive.profile.j2" dest="{{ ngi_pipeline_conf }}/texlive.profile" mode=0664
   tags: ngi_reports 
 
+- shell: "find {{ texLive_dest }}/src/install* -maxdepth 0 -type d"
```

Comment:
Did something similar which exploits the naming convention rather than modification date. The LaTeX installation process is awful. Good catch!